spec/controllers/automated_tests_controller_spec.rb

```diff
@@ -129,12 +129,41 @@
     end
     context 'GET download_files' do
       subject { get_as role, :download_files, params: params }
-      let(:content) { response.body }
-      it_behaves_like 'zip file download'
```

Comment:
You should not delete existing tests unless there's a good reason. I'm don't see one in this case.